maps/009-1.tmx

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<map version="1.4" tiledversion="1.4.1" orientation="orthogonal" renderorder="right-down" compressionlevel="0" width="117" height="75" tilewidth="32" tileheight="32" infinite="0" nextlayerid="11" nextobjectid="95">
+<map version="1.2" tiledversion="1.2.1" orientation="orthogonal" renderorder="right-down" width="117" height="75" tilewidth="32" tileheight="32" infinite="0" nextlayerid="11" nextobjectid="95">
```

Comment:
(Your Tiled is outdated)

Reply:
its was came with debian i saw that there was a new version but since most maps where version="1.0" or so I don't thought its needed.

Reply:
Not map version, Tiled Version :p